apps/communications/contacts/views/form/js/boot.js

```diff
@@ -0,0 +1,59 @@
+/* global LazyLoader, FormUI, Form */
+'use strict';
+
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-form')
+    ], function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+
+window.addEventListener('load', function() {
+  var dependencies = [
+    '/contacts/js/navigation.js',
+    '/contacts/services/contacts.js',
+    '/shared/js/l10n_date.js',
+    '/shared/js/contact_photo_helper.js',
+    '/shared/js/contacts/utilities/templates.js',
+    '/contacts/js/contacts_tag.js',
+    '/contacts/js/tag_options.js',
+    '/shared/js/text_normalizer.js',
+    '/shared/js/contacts/import/utilities/status.js',
+    '/shared/js/contacts/utilities/dom.js',
+    '/shared/js/contacts/import/utilities/misc.js'
+  ];
+
+  LazyLoader.load(
+    dependencies,
+    function() {
+      LazyLoader.load(
+        [
```

Comment:
+1. This was a race condition before, but now everything works as expected. Thanks!